web-server/v0.4/src/app.js

```diff
@@ -6,6 +6,7 @@ const persistConfig = {
   timeout: 1000,
   key: 'root',
   storage,
+  blacklist: ['dashboard'],
```

Comment:
What does it mean to "blacklist" exactly?  Can you add a comment describing why the `dashboard` data is black listed here?

Reply:
blacklisting allows us to prevent certain redux store namespaces from being persisted. In this case, we want to avoid persisting the dashboard namespace as server side information is constantly changing and we avoid pulling new data if the information has already been persisted.

Reply:
I've also added the search namespace to this config as the search view renders information that is dependent on server side information being modified or updated.